airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -216,11 +217,14 @@ def run_query(
         :type udf_config: list
         :param use_legacy_sql: Whether to use legacy SQL (true) or standard SQL (false).
         :type use_legacy_sql: boolean
+        :param maximum_billing_tier: Positive integer that serves as a multiplier of the basic price.
+        :type maximum_billing_tier: integer
         """
         configuration = {
             'query': {
                 'query': bql,
-                'useLegacySql': use_legacy_sql
+                'useLegacySql': use_legacy_sql,
+                'maximumBillingTier': maximum_billing_tier
```

Comment:
Hi Chris, just tested it. It works just fine with None as default argument. This way it uses the project default (1 in my case). Using False as a default argument will not work. Thanks!